### PR TITLE
Pavel/RTE update: maxchars + inline styles

### PR DESCRIPTION
**Pull request by galloween**
1) maxchars implementation (with test!)
2) align, direction & font-size now use inline styles, not classes
3) quill styles are now scoped to RTE component

### CODE REVIEW

File: projects/ui-framework/src/lib/style/quill/quill-sizes.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     font-size: 0.75em;
   }
-  .ql-size-large {
+  #{$ql-size-large} {
     font-size: 1.5em;
   }
-  .ql-size-huge {
+  #{$ql-size-huge} {
     font-size: 2.5em;
   }
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/style/quill/quill-sizes.scss]: why not pixels like you set inline?

File: projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts [hunk 2]
```diff
@@ -60,6 +64,16 @@ export abstract class RTEformElement extends BaseFormElement
...
 
+  @HostBinding('class.length-invalid') get isLengthInvalid(): boolean {
+    return (
+      this.length < this.minChars ||
+      (this.maxChars && this.length > this.maxChars)
+    );
+  }
+  @HostBinding('class.length-warning') get hasLengthWarning(): boolean {
+    return this.maxChars && this.maxChars - this.length < 15;
+  }
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts]: minChars is oblivious to dom markup?

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts]: did someone ask for this? if not, you can either remove, or if you want to keep, add tests

Comment (galloween) [reply projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts]: somebody asked, yes. the counter only considers 'plain text' length.

File: projects/ui-framework/src/lib/style/quill/quill-sizes.scss (hunk 1 above)
Comment (galloween) [reply projects/ui-framework/src/lib/style/quill/quill-sizes.scss]: that was original quill style, i just changed the selector.
the pixels values are also quill defaults.

File: projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts (hunk 2 above)
Comment (galloween) [reply projects/ui-framework/src/lib/form-elements/rich-text-editor/rte-core/rte-form-element.abstract.ts]: will remove till they ask again. not the most important feature.

**Pull request merged**